src/redux/initialState.js

```diff
@@ -976,8 +976,39 @@ const initialState = {
   compare: {
     products: [],
   },
+  promotedProducts: {
+    firstProduct: {
```

Comment:
To generalnie powinna być tabela produktów, ale dlaczego nie dodałaś do produktu właściwośći w rodzaju `hotDeals` i pobrać tylko element z tą flagą do komponentu.